Oracle/PCORNetLoader_ora.sql

```diff
@@ -415,66 +415,6 @@ begin
 end;
 /
 
-BEGIN

```

Comment:
@njgraham , @mprittie noted this is a somewhat invasive change to the SCILHS code but he said you and he chatted about it and couldn't think of a better idea. Does that sound about right?